packages/edit-site/src/store/selectors.js

```diff
@@ -71,7 +71,17 @@ export const __experimentalGetPreviewDeviceType = createRegistrySelector(
  * @return {Object} Whether the current user can create media or not.
  */
 export const getCanUserCreateMedia = createRegistrySelector(
-	( select ) => () => select( coreDataStore ).canUser( 'create', 'media' )
+	( select ) => () => {
+		deprecated(
+			`wp.data.select( 'core/edit-site' ).getCanUserCreateMedia()`,
+			{
+				since: '6.7',
+				alternative: `wp.data.select( 'core' ).canUser()`,
```

Comment:
Should we be more specific here?

Reply:
We have mixed usage of alternative suggestions. So not sure 😅

Reply:
What do you mean? Here the only alternative is `canUser( 'create', 'media' )`, no?

Reply:
Got it. Sure, I'll update the suggestion.